### DISCUSSION TITLE
GitHub sponsors support request for Botswana

### DISCUSSION THREAD

**Discussion by kevstar0** [community/community#15360] [Other Feature Feedback, Questions, & Ideas]
Hello everyone, I know I am likely to be the only person to mention this but I plead to GitHub to add Botswana as one of the countries they'd invest/ sponsor reason being is although I teach others about GitHub and it's usefulness trying to get more users for it with the goal of them to learn about open source projects to contribute & learn new tech I wish to see my country move from their stone age era into a futuristic one. I am no president but I speak for all 2million of us that we need investors...

FACTS!!!

without the intention of trying to upset anyone but the truth is in Botswana not even Youtubers can get paid by youtube/ google ad-sense due to unknown factors between us (Botswana & Google) and it was also came to me as a surprise to realize GitHub did the same

Is it intentional? I wouldn't know, but to just earnestly ask for support

But I thank GitHub for allowing me to host my website here, it's an honour to be part of the community 💕❤️

here are my Youtube & GitHub support findings
![image](https://user-images.githubusercontent.com/38152088/164549222-7dcacdda-7fb5-485e-b12a-f06fc923eed3.png)
![image](https://user-images.githubusercontent.com/38152088/164548670-a49479ff-b7df-40f0-a1b0-c02bce1d1782.png)

**Comment by kevstar0** [comment]
BUT!
weirdly enough our neighbouring country South Africa is eligible on both sides

are we blacklisted or????

I need answers😭

**Reply by pythoncrazy** [reply]
I fully support this measure
